EpsilonDriverControls/Inc/DriverControls.h

```diff
@@ -44,6 +44,9 @@
 #define M0_VELOCITY_INPUT_STDID 0x403U
 #define M1_VELOCITY_INPUT_STDID 0x423U
 
+#define SAFE_VEHICLE_VELOCITY_TO_GO_FORWARD 0.0f // Will probably be 0 or a small negative number
+#define SAFE_VEHICLE_VELOCITY_TO_GO_REVERSE 0.0f // Will probably be 0 or a small positive number
```

Comment:
Maybe give a bit of buffer, I don't know how it will react on slow turns from rest - eg if you turn right, the left motor goes forward but the right one might stay staionary or move backward a bit?

Reply:
Do you have an idea of a good number? Maybe like +/- 10 km/h?